src/backend/models/match.ts

```diff
@@ -217,6 +218,7 @@ export class Match {
 
         return {
             ID: this.metadata.matchId,
+            Map: ['ARAM', 'KINGPORO'].includes(this.info.gameMode) ? MatchMap.HA : MatchMap.SR,
```

Comment:
Should we have a mapping somewhere between game modes and maps instead?

Reply:
We should, but this only affects the win/loss icon in the match list for now, so not very important.
Let's see if we can find an official mapping because I'm not sure which other game modes should have the single lane icon.